tests/integration/test_get_deposits.py

```diff
@@ -423,3 +432,212 @@ def test_get_deposit_with_permissions_json_serializer_returns_serialized_permiss
+    db.session.add(webhook)
+    db.session.commit()
+
+    # get the deposit
+    #################
+    pid = deposit['_deposit']['id']
+    metadata = deposit.get_record_metadata()
+    bucket = deposit.files.bucket
+    headers = auth_headers_for_example_user + [('Accept', 'application/form+json')]
+
+    resp = client.get(f'/deposits/{pid}', headers=headers)
+
+    assert resp.status_code == 200
+    assert resp.json == {
+        'access': {
+            'deposit-admin': {'roles': [], 'users': [example_user.email]},
+            'deposit-read': {'roles': [], 'users': [example_user.email]},
+            'deposit-update': {'roles': [], 'users': [example_user.email]}
+        },
+        'created_by': example_user.email,
+        'is_owner': True,
+        'can_admin': True,
+        'can_update': True,
+        'experiment': 'CMS',
+        'created': metadata.created.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
+        'updated': metadata.updated.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
+        'id': pid,
+        'links': {
+            'bucket': f'http://analysispreservation.cern.ch/api/files/{str(bucket)}',
+            'clone': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/clone',
+            'discard': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/discard',
+            'edit': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/edit',
+            'files': f'http://analysispreservation.cern.ch/api/deposits/{pid}/files',
+            'html': f'http://analysispreservation.cern.ch/drafts/{pid}',
+            'permissions': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/permissions',
+            'publish': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/publish',
+            'self': f'http://analysispreservation.cern.ch/api/deposits/{pid}',
+            'upload': f'http://analysispreservation.cern.ch/api/deposits/{pid}/actions/upload'
+        },
+        'metadata': {}, 'revision': 1, 'status': 'draft', 'type': 'deposit',
+        'labels': [], 'workflows': [],
+        'schema': {'name': 'test-schema', 'version': '1.0.0'},
+        'schemas': {
+            'schema': {
+                'title': 'deposit-test-schema',
+                'type': 'object',
+                'properties': {
+                    'date': {'type': 'string'},
+                    'title': {'type': 'string'}},
+                },
+            'uiSchema': {
+                'title': 'ui-test-schema',
+                'type': 'object',
+                'properties': {
+                    'field': {'type': 'string'},
+                    'title': {'type': 'string'}},
+                }
+        },
+        'files': [],
+        'webhooks': [{
+            'branch': 'mybranch',
+            'event_type': 'push',
+            'snapshots': []
+        }]
+    }
+
+
+@responses.activate
+@patch.object(Github, 'get_repo')
+def test_get_deposit_with_form_json_serializer_with_files(
```

Comment:
like discussed, this one is unnecessary (we don't need to test git upload here, and files, webhooks and snapshots are already covered in one above)